server/src/test/scala/services/crunch/ArrivalUpdatesCorrectlyAffectLoads.scala

```diff
@@ -0,0 +1,179 @@
+package services.crunch
+
+import controllers.ArrivalGenerator
+import drt.shared.{Arrival, DqEventCodes, Queues}
+import drt.shared.CrunchApi.PortState
+import drt.shared.FlightsApi.Flights
+import passengersplits.parsing.VoyageManifestParser.{PassengerInfoJson, VoyageManifest}
+import server.feeds.{ArrivalsFeedSuccess, ManifestsFeedSuccess}
+import services.SDate
+import services.crunch.VoyageManifestGenerator.{euPassport, nonVisa, visa, euIdCard}
+import services.graphstages.DqManifests
+
+import scala.collection.immutable.List
+import scala.concurrent.duration._
+
+class ArrivalUpdatesCorrectlyAffectLoads extends CrunchTestLike {
+  val crunch: CrunchGraphInputsAndProbes = runCrunchGraph(
+    now = () => SDate("2019-01-01T01:00"),
+    pcpArrivalTime = pcpForFlightFromBest,
+    airportConfig = airportConfig.copy(
+      queues = Map("T1" -> Seq(Queues.EeaDesk, Queues.NonEeaDesk, Queues.EGate), "T2" -> Seq())
+    )
+  )
+  val arrivalOne: Arrival = ArrivalGenerator.arrival(
+    iata = "BA0001",
+    terminal = "T1",
+    origin = "JFK",
+    schDt = "2019-01-01T00:00",
+    actPax = Option(100)
+  )
+
+  val arrivalTwo: Arrival = ArrivalGenerator.arrival(
+    iata = "BA0002",
+    terminal = "T1",
+    origin = "JFK",
+    schDt = "2019-01-01T00:05",
+    actPax = Option(117)
+  )
+
+  "Given crunch inputs and an arrival" >> {
+
+    "I should see the correct number of pax along with the right minutes affected" >> {
+
+      "When I send in a new arrival with only a scheduled date set" >> {
+        offerAndCheckResult(Seq(arrivalOne))
+        success
+      }
+    }
+
+    "When I send an updated estimated time" >> {
+      val updatedArrival = arrivalOne.copy(
+        Estimated = Option(SDate("2019-01-01T00:07").millisSinceEpoch)
+      )
+
+      offerAndCheckResult(Seq(updatedArrival))
+      success
+    }
+
+    "When I send an updated act pax" >> {
+      val updatedArrival = arrivalOne.copy(
+        Estimated = Option(SDate("2019-01-01T00:07").millisSinceEpoch),
+        ActPax = Option(105)
```

Comment:
i there some significance to these pax numbers or are they as random as they look?

Reply:
If there is a reason for them, might be worth naming them in a variable that conveys the meaning, if not it might make sense to make them all the same?

Reply:
They are random, but purposely different from each other to simulate updated pax numbers coming through from a feed

Reply:
👍